livekit-plugins/livekit-plugins-google/livekit/plugins/google/log.py

```diff
@@ -1,3 +1,6 @@
 import logging
 
 logger = logging.getLogger("livekit.plugins.google")
+
+
+logging.getLogger("websockets.client").setLevel(logging.INFO)
```

Comment:
```suggestion
```

Reply:
This is to suppress verbose websocket logs

Reply:
This is too opiniated to be inside this place, you can add it here instead https://github.com/livekit/agents/blob/32383939025453616d23c82ccc8c725ee87a9870/livekit-agents/livekit/agents/cli/log.py#L15